CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewPane.java

```diff
@@ -429,9 +486,15 @@ protected synchronized void updateNotesUI() {
             synchronized (LOCK) {
                 notesViewEntries.forEach(entry -> {
                     // Add note to render list if its respective filter is selected.
-                    if ((selectedFilters.contains(USER_NOTES_FILTER) && entry.isUserCreated())
-                            || (selectedFilters.contains(AUTO_NOTES_FILTER) && !entry.isUserCreated())) {
+                    if ((selectedFilters.contains(USER_NOTES_FILTER) && entry.isUserCreated())) {
                         notesToRender.add(entry);
+
+                    } else if (selectedFilters.contains(AUTO_NOTES_FILTER) && !entry.isUserCreated()) {
+                        final boolean addNote = updateAutoNotesDisplayed(entry);
+                        if (addNote) {
```

Comment:
I'd probably just merge these together given this is the only spot the boolean expression is used